packages/jest-preset-mc-app/transform-babel-jest.js

```diff
@@ -1,7 +1,10 @@
 const getBabePresetConfigForMcApp = require('@commercetools-frontend/babel-preset-mc-app');
 const getJestBabelPreset = require('babel-preset-jest');
+const hasJsxRuntime = require('./has-jsx-runtime');
 
-const mcAppBabelConfig = getBabePresetConfigForMcApp();
+const mcAppBabelConfig = getBabePresetConfigForMcApp(null, {
```

Comment:
We spread in the `babelJestConfig` we we might have to do it this way here if we don't want to change that.